script.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+# Check if `git` exists in the current env.
+if ! command -v git &> /dev/null; then
+    echo "git could not be found"
+    exit 2
+fi
+
+ERRORS=`git log --grep='^\w+(\((\w|-)+\))?!?: .+$' --invert-grep -E --pretty=format:%s | awk 'END {print NR}'`
```

Comment:
Usage of legacy backticks: use `"$(...)"` instead.

Reply:
Thank you, I'll use the newer notation